sklearn/utils/estimator_checks.py

```diff
@@ -403,6 +404,40 @@ def pairwise_estimator_convert_X(X, estimator, kernel=linear_kernel):
     return X
 
 
+def _generate_sparse_matrix(X_csr):
+    """Generate matrices in multiple formats for CSR,CSC and COO matrices
```

Comment:
I know it's a private function, but this could be reformulated to,
```
Generate 32 and 64 bit indexed sparse matrices in CSR, CSC and COO format
```